src/config/constants.ts

```diff
@@ -82,9 +82,9 @@ export const FREIGHTER_GITHUB_ISSUE_URL =
   "https://github.com/stellar/freighter-mobile/issues";
 export const STELLAR_FOUNDATION_BASE_URL = "https://stellar.org";
 export const STELLAR_FOUNDATION_TERMS_URL =
```

Comment:
@Keeeeeeeks thanks for taking the stab on it!

Would you be able to update all references of `STELLAR_FOUNDATION_TERMS_URL` to `FREIGHTER_TERMS_URL`, and all references of `STELLAR_FOUNDATION_PRIVACY_URL` to `FREIGHTER_PRIVACY_URL` in the project so it better reflects the content of the new urls?

Reply:
FYI, I've updated the references on the other files here: https://github.com/stellar/freighter-mobile/pull/508/commits/ee19964fe4627543aa82d3e22ea55e73a3dedde5